pallets/liquidity-pools-gateway/src/tests.rs

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
TODO: pending

Reply:
@cdamian, are you in charge of this TODO, right?

Reply:
Yup, they're green as of now in the [multi-router PR](https://github.com/centrifuge/centrifuge-chain/pull/1961).